boards/opentitan/earlgrey-cw310/run.sh

```diff
@@ -24,8 +24,8 @@ if [[ "${VERILATOR}" == "yes" ]]; then
 		--meminit=flash,./"$BUILD_DIR"/binary.64.vmem \
 		--meminit=otp,${OPENTITAN_TREE}/build-out/sw/device/otp_img/otp_img_sim_verilator.vmem
 elif [[ "${OPENTITAN_TREE}" != "" ]]; then
-	${OBJCOPY} --update-section .apps=${APP} ${1} bundle.elf
-	${OBJCOPY} --output-target=binary bundle.elf binary
+	riscv64-linux-gnu-objcopy --update-section .apps=${APP} ${1} bundle.elf
```

Comment:
We could probably do the same for the Verilator and get rid of OBJCOPY altogether?

Reply:
If the LLVM one works it's probably best to use that (I just can't get it working here)